src.gen/@amzn/codewhisperer-streaming/package.json

```diff
@@ -20,15 +20,15 @@
         "tslib": "^2.5.0",
         "@aws-crypto/sha256-browser": "3.0.0",
         "@aws-crypto/sha256-js": "3.0.0",
-        "@aws-sdk/middleware-host-header": "3.489.0",
-        "@aws-sdk/middleware-logger": "3.489.0",
-        "@aws-sdk/middleware-recursion-detection": "3.489.0",
-        "@aws-sdk/middleware-token": "3.489.0",
-        "@aws-sdk/middleware-user-agent": "3.489.0",
-        "@aws-sdk/region-config-resolver": "3.489.0",
-        "@aws-sdk/types": "3.489.0",
-        "@aws-sdk/util-user-agent-browser": "3.489.0",
-        "@aws-sdk/util-user-agent-node": "3.489.0",
+        "@aws-sdk/middleware-host-header": "3.425.0",
```

Comment:
The generator is actually *reverting* dependency updates that were done by dependabot. @hayemaxi @nkomonen-amazon 

We're currently trying to decide if we can/should tell dependabot to ignore this generated subproject. But I think it probably makes sense to un-revert these dependencies in this PR. That should be simply a matter of reverting all changes to these files:

```
src.gen/@amzn/codewhisperer-streaming/package.json
src.gen/@amzn/codewhisperer-streaming/package-lock.json
```

Reply:
The newly generated client is not compatible with versions `3.489.0`. Unless we're willing to manually fix problems whenever those are updated, it might be best to keep them at the auto generated versions.

Reply:
Hmm... ok.